third_party/move/testing-infra/test-generation/src/lib.rs

```diff
@@ -58,8 +54,15 @@ fn run_verifier(module: CompiledModule) -> Result<CompiledModule, String> {
     }
 }
 
-static STORAGE_WITH_MOVE_STDLIB: Lazy<InMemoryStorage> = Lazy::new(|| {
+// Creates a storage with Move standard library as well as a single
+// additional module.
+fn storage_with_stdlib_and_module(
+    additional_module_id: ModuleId,
```

Comment:
Why not a vec of module ids and blobs? seems more general (esp. the fact that can also provide no additional) at barely any code invoncenience

Reply:
Previous code was publishing a single module, so implemented it the same way. Change to take a vector of compiled modules instead👍